scripts/utils/createJestConfig.js

```diff
@@ -9,14 +9,22 @@
  */
 // @remove-on-eject-end
 
+const pathExists = require('path-exists');
+const paths = require('../../config/paths');
+
 module.exports = (resolve, rootDir) => {
+  const setupFiles = [resolve('config/polyfills.js')];
+  if (pathExists.sync(paths.testsSetup)) {
+    setupFiles.push(paths.testsSetup);
+  }
+
   const config = {
     moduleNameMapper: {
       '^[./a-zA-Z0-9$_-]+\\.(jpg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm)$': resolve('config/jest/FileStub.js'),
       '^[./a-zA-Z0-9$_-]+\\.css$': resolve('config/jest/CSSStub.js')
     },
     scriptPreprocessor: resolve('config/jest/transform.js'),
-    setupFiles: [resolve('config/polyfills.js')],
+    setupFiles: setupFiles,
```

Comment:
True, wasn't sure of what es6 features I could use there since there are some "var" elsewhere in the code :s